karma.conf.js

```diff
@@ -18,10 +18,11 @@ module.exports = function(config) {
 			'jspm_packages/system-polyfills.js',
 			'jspm_packages/system.src.js',
 			'jspm.config.js',
-			{ pattern: 'spec/**/*.spec.build.js', watched: true, included: true, served: true },
-			{ pattern: 'spec/**/*.*', watched: true, included: false, served: true },
+			'spec/bundles/webpack2.spec.build.js',
```

Comment:
Now that webpack2 is easily supported, I added a whole new test suite for webpack that reuses the tests that were in the systemjs bundle